GNUmakefile

```diff
@@ -20,13 +20,14 @@ GOFILES_NOVENDOR = $(shell find . -type f -name '*.go' -not -path "./vendor/*")
 # Specify the name for the binaries
 MAYACTL=mayactl
 APISERVER=maya-apiserver
-AGENT=maya-agent
+NODEBOT=maya-nodebot
```

Comment:
Should the NODEBOT change be included in this PR?

Reply:
ok, will remove nodebot, will rename cstorcrd to cstorPool, will make name change of sidecar in a separate pr